NSS-Day-Cohort-42/rare-media-moguls
# Ap subscriptions

#### Changes Made
1. Modified files `SubscribeButton.js` and `SubscriptionProvider.js` to carry out subscribe and unsubscribe functions based on current subscription status

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout ap-subscriptions
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> Pull down and see instructions at `https://github.com/NSS-Day-Cohort-42/rare-API-media-moguls/pull/44`
5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
